shared_requirements.txt

```diff
@@ -534,4 +534,24 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-mgmt-resourcegraph msrest>=0.7.1
 #override azure-core-experimental azure-core<2.0.0,>=1.25.0
 #override azure-developer-devcenter isodate<1.0.0,>=0.6.1
-#override azure-developer-devcenter azure-core<2.0.0,>=1.24.0
\ No newline at end of file
+#override azure-developer-devcenter azure-core<2.0.0,>=1.24.0
+
+# azure-ai-ml
+pyyaml<7.0.0,>=5.1.0
+msrest>=0.6.18
+azure-core<2.0.0,>=1.8.0, !=1.22.0
+azure-mgmt-core<2.0.0,>=1.3.0
+marshmallow<4.0.0,>=3.5
+jsonschema<5.0.0,>=4.0.0
+tqdm<=4.63.0
+strictyaml<=1.6.1
```

Comment:
Please update this to a minimum version only. If you must have an upper bound - it must be the next non-current major version.

Reply:
This applies to a few other packages as well.